websocket-chat-broker/README.md

```diff
@@ -0,0 +1,29 @@
+# Websocket chat broker example
+
+This is a rewrite of the
```

Comment:
Let's say it is different implementation of websocket chat example

Reply:
Changed to "different implementation"